charts/node/README.md

```diff
@@ -70,6 +70,25 @@ node:
 
 ## Upgrade
 
+### v4.6.0 (⚠️ breaking change)
+
+Substrate changed the default rpc flags: https://github.com/paritytech/substrate/pull/13384 \
+Port `9933` was replaced by combined port `9944`
+Flags replaced:
+```
+--rpc-max--payload (replaced by --rpc--max-request-size and --rpc-max-response-size)
+--ws-max-out-buffer-capacity (removed)
+--ws-external (replaced by --rpc-external)
+--unsafe-ws--external (replaced by --unsafe-rpc-external)
+--ipc-path (removed)
+--ws-port (replaced by --rpc-port)
+--ws-max-connections (replaced by --rpc-max-connections)
+--rpc-http (replaced by --rpc-addr)
+--rpc-ws (replaced by --rpc-addr)
+```
+New value was added to support this change:
+- `node.legacyRpcFlags`
```

Comment:
Add the following comment, if your node is still using the old RPC flags, please set `node.legacyRpcFlags=true`